### PR TITLE
Legal reference for Nebraska nonrefundable CDCC and social security AGI reduction

### PR DESCRIPTION
**Pull request by universescraper**
Added legal reference for non-refundable child and dependent care credit and Adjusted gross income amount reduced by Social Security Benefit.

### CODE REVIEW

**Review by MaxGhenis — COMMENTED** [review]
Thanks could you just run `make format`?

File: changelog_entry.yaml [hunk 1]
```diff
@@ -1,4 +1,20 @@
 - bump: patch
   changes:
     added:
-    - Legal reference for NE CDCC and EITC.
\ No newline at end of file
+    - Legal reference for NE CDCC and EITC.
```
Comment (MaxGhenis) [inline changelog_entry.yaml:4]: Remove everything below this and fix what looks like a merge conflict via `git pull upstream master`
```suggestion
    - Legal reference for Nebraska CDCC, EITC, AGI reduction by SSB, and extra standard deduction.
```

Comment (universescraper) [reply changelog_entry.yaml]: Hi Max, what does make format do? I typed the command on github and all it says is "black . -l 79"

Also, I am assuming git pull upstream master updates my current codespace from the original source. I am not 100% sure though. Do you have any recommendations when it comes to figuring out how exactly codespace works?

File: policyengine_us/parameters/gov/states/ne/tax/income/deductions/standard/base_amount.yaml [hunk 2]
```diff
@@ -10,7 +10,7 @@ metadata:
     href: https://revenue.nebraska.gov/files/doc/tax-forms/2021/f_1040n_booklet.pdf
   - title: 2022 NE income tax form and instruction booklet
     href: https://revenue.nebraska.gov/files/doc/2022_Ne_Individual_Income_Tax_Booklet_8-307-2022_final_5.pdf
-
+  - title: Nebraska Revised Statute | chapter 77 | 2716.01
```
Comment (MaxGhenis) [inline policyengine_us/parameters/gov/states/ne/tax/income/deductions/standard/base_amount.yaml:13]: needs href

File: changelog_entry.yaml (hunk 1 above)
Comment (MaxGhenis) [reply changelog_entry.yaml]: `make format` formats the code via `black` and `linecheck` - you can see what it does in `Makefile`

codespaces is an IDE, but yes `git pull upstream master` syncs your branch with the policyengine/policyengine-us master branch. looks like you still have a merge conflict in changelog_entry.yaml, which you should be able to resolve by clicking "accept current" when opening that file

**Comment by MaxGhenis** [comment]
Please run make format

**Comment by MaxGhenis** [comment]
There are still missing linebreaks. If you're running locally make sure to run `make install` then `make format`.

Please also try to use descriptive commit messages to make it easier for reviewers to know what to look for. If it's not ready for review, feel free to mark it as draft until it is.

**Comment by universescraper** [comment]
> 

so when I do "make format" my codespaces says 

black . -l 79
All done! ✨ 🍰 ✨
1326 files left unchanged.
linecheck . --fix
make: linecheck: Command not found
make: *** [Makefile:4: format] Error 127

I am not sure how I can resolve this error and if it affects the ability to format.

Also, I just noticed when I type "conda activate policyengine" it says

conda activate policyengine

CommandNotFoundError: Your shell has not been properly configured to use 'conda activate'.
To initialize your shell, run

    $ conda init <SHELL_NAME>

Currently supported shells are:
  - bash
  - fish
  - tcsh
  - xonsh
  - zsh
  - powershell

See 'conda init --help' for more information and options.

IMPORTANT: You may need to close and restart your shell after running 'conda init'.

I will add more descriptions in the commit messages!

**Comment by MaxGhenis** [comment]
Did you run `make install` first?

**Comment by MaxGhenis** [comment]
Finished and merged as I needed to update the API for debugging and this was the closest open PR.

**Pull request merged**